### PR TITLE
fixed strings

### PR DESCRIPTION
**Pull request by harshkpatel**
Fixed the Issue richard created, nothing changed in the logic

### CODE REVIEW

**Review by benxu3 — APPROVED** [review]
Good catch!

**Review by kRichard32 — APPROVED** [review]
Good fix, nothing else to say.

File: src/initialoutputs/states.txt [hunk 1]
```diff
@@ -11,8 +11,8 @@ Main Tree Parent Node,""
 Trade,Who do you want to tradeUseCase with?
```
Comment (giaphuongd) [inline src/initialoutputs/states.txt:11]: maybe replace tradeUseCase with trade here?

File: src/initialoutputs/states.txt [hunk 2]
```diff
@@ -11,8 +11,8 @@ Main Tree Parent Node,""
...
 Pick Player (Trade),What property do you want from the player?
 Trader/Tradee Has No Properties,Trade cannot be done; one of you have no properties
```
Comment (giaphuongd) [inline src/initialoutputs/states.txt:13]: Not a big deal but capitalization is a little inconsistent here

**Review by giaphuongd — CHANGES_REQUESTED** [review]
I think you missed one line. If you intend to keep tradeUseCase then ignore this change request because the rest looks good. If you have time, maybe fix it so that capitalization is consistent too, but that's not super important.

File: src/initialoutputs/states.txt (hunk 1 above)
Comment (harshkpatel) [reply src/initialoutputs/states.txt]: Addressed

File: src/initialoutputs/states.txt (hunk 2 above)
Comment (harshkpatel) [reply src/initialoutputs/states.txt:13]: The capitalization is inconsistent with the key of the hashmap, and if I change the key value, its going to be very tedious to trace through the entire program to find where this use case is use.

**Review by giaphuongd — APPROVED** [review]

**Pull request merged**